core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -40,13 +40,30 @@ oppia.directive('topNavigationBar', [
           $scope.NAV_MODE = GLOBALS.NAV_MODE;
           $scope.LABEL_FOR_CLEARING_FOCUS = LABEL_FOR_CLEARING_FOCUS;
           $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
-
+          
+          $scope.activeMenuName = '';
           $scope.username = GLOBALS.username;
           $scope.profilePictureDataUrl = GLOBALS.profilePictureDataUrl;
           $scope.isAdmin = GLOBALS.isAdmin;
           $scope.isModerator = GLOBALS.isModerator;
           $scope.isSuperAdmin = GLOBALS.isSuperAdmin;
           $scope.logoutUrl = GLOBALS.logoutUrl;
+          $scope.ACTION_OPEN = 'open';
```

Comment:
Probably best to use these in the HTML template as well.

Reply:
Done.

Reply:
I still see 'open' and 'close' all over the place in that file...

Reply:
Done.In case of the class name we cannot use ACTION_OPEN, instead, I have written in this way(<[ACTION_OPEN>])

Reply:
Oh, sorry -- I think this is my bad, actually. The class name 'open'/'close' isn't an action, right? It's just a communication between the HTML and the CSS, so you can leave that as-is.

I really meant to comment on ACTION_CLOSE. E.g. in line 29 it's really weird to have one of the values be ACTION_OPEN and the other to just be 'close'.

Reply:
oh sorry, I thought you are suggesting to change the class name, I'll fix it.
Thanks